workers/unity/Packages/com.improbable.gdk.core/Commands/WorldCommands.cs

```diff
@@ -700,11 +700,6 @@ public struct Request
 
             public static Request CreateRequest(Improbable.Worker.Query.EntityQuery entityQuery, uint? timeoutMillis = null, Object context = null)
             {
-                if (entityQuery.ResultType is SnapshotResultType)
-                {
-                    Debug.LogWarning("Cannot safely access component data from entity query - this is a known issue. To protect its integrity, the worker will drop the request before sending.");
```

Comment:
perhaps we could leave the warning in, but let the user know that only the entity ids will be returned and not the component data

Reply:
It'd be too spammy imo.

I would make the warning optional somehow... like a `bool iKnowWhatIAmDoing`, or 

`#if DISABLE_WARNING_ENTITY_QUERIES_CAN_ONLY_RETURN_ENTITY_IDS`

Or a `CreateRequestForEntityIds` next to it that will omit the warning, or something like that?

Reply:
Why remove this. This seems correct.